frontend/src/components/image/styles.scss

```diff
@@ -0,0 +1,22 @@
+.Image {
+  width: 100%;
+  height: 100%;
+  flex-grow: 1;
+  font-size: 2rem;
+  line-height: 2rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+
+  &-image {
+    width: 100%;
+    z-index: 1;
```

Comment:
Any specific reason for `z-index`?

Reply:
It's to avoid the loading indicator (theoretically) briefly overlaying the image. It takes a rendering cycle for the loader to get removed after the image is done.

Reply:
I see. I've managed to get the desired element stacking without z-index, with the help of [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_without_z-index).
(relative-positioned element over absolute-positioned)
Tested for both scene and performer images.

```diff
diff --git a/frontend/src/components/image/Image.tsx b/frontend/src/components/image/Image.tsx
index 45de6e0..bf9e2a7 100644
--- a/frontend/src/components/image/Image.tsx
+++ b/frontend/src/components/image/Image.tsx
@@ -28,13 +28,6 @@ const Image: FC<Props> = ({
 
   return (
     <>
-      <img
-        alt=""
-        src={url}
-        className={`${CLASSNAME}-image`}
-        onLoad={() => setImageState("done")}
-        onError={() => setImageState("error")}
-      />
       {imageState === "loading" && (
         <LoadingIndicator message="Loading image..." delay={200} />
       )}
@@ -46,6 +39,13 @@ const Image: FC<Props> = ({
           <span>Failed to load image</span>
         </div>
       )}
+      <img
+        alt=""
+        src={url}
+        className={`${CLASSNAME}-image`}
+        onLoad={() => setImageState("done")}
+        onError={() => setImageState("error")}
+      />
     </>
   );
 };
diff --git a/frontend/src/components/image/styles.scss b/frontend/src/components/image/styles.scss
index fa7abd6..e3e92b7 100644
--- a/frontend/src/components/image/styles.scss
+++ b/frontend/src/components/image/styles.scss
@@ -8,11 +8,10 @@
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  position: relative;
 
   &-image {
+    position: relative;
     width: 100%;
-    z-index: 1;
   }
 
   .LoadingIndicator {
```